core/src/mindustry/input/InputHandler.java

```diff
@@ -499,11 +500,40 @@ protected void showSchematicSave(){
                 });
             }else{
                 lastSchematic.tags.put("name", text);
+                lastSchematic.tags.put("description", description);
                 schematics.add(lastSchematic);
                 ui.showInfoFade("@schematic.saved");
                 ui.schematics.showInfo(lastSchematic);
             }
-        });
+        };
+
+        new Dialog("@schematic.add"){{
```

Comment:
Is it *really* important for the player to get a description entry dialog when saving the schematic for the first time? If a player cares, they can add it in later. Could be annoying to have an extra field pop up every time you want to save a small utility schematic.

Reply:
Yes, you are right. I'll fix it soon.